### DISCUSSION TITLE
UG Native IOS PropTypes Error

### DISCUSSION THREAD

**Discussion by pdorozhkin** [JiLiZART/BBob#177] [Q&A]
![image](https://github.com/JiLiZART/BBob/assets/134928280/c36bb5d6-385b-41cc-8ae3-556f859cd9fd)

**Comment by JiLiZART** [comment]
This package didn't install `prop-types` package for you.
It's a peer dep for `react` preset. You need install `prop-types` manualy

```
"peerDependencies": {
    "prop-types": "> 15.0",
    "react": "> 15.0"
  },
```

**Comment by a-sobolenkov-muse** [comment]
We use TypeScript so we don't need a 'prop-types' package.
Properties checks are now performed at compile time.

Is it posible to make 'prop-types' optional for `bbob-react`?

**Comment by JiLiZART** [comment]
`bbob-react` doesn't use TypeScript, for React prop checking it uses `prop-types` package, but don't lock version of `prop-types` therefore `prop-types` is a peer deps package (you need install it by yourself). This approach excludes package dublication if some other packages already use `prop-types`.
